src/components/stop-event-propagation.js

```diff
@@ -1,6 +1,5 @@
 const handler = e => {
   e.stopPropagation();
-  e.preventDefault();
```

Comment:
Fixes issues where when interacting with UI we end up with messed up superhands state.